pkgs/test_api/lib/scaffolding.dart

```diff
@@ -2,6 +2,12 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+/// {@canonicalFor on_platform.OnPlatform}
```

Comment:
![image](https://user-images.githubusercontent.com/17034/148624215-41377970-2fc9-40d4-81d3-ffbfa283ae1c.png)

Reply:
I think we're still in a place where we expect most folks to get these through `package:test/test.dart`, and if they didn't get it from there I think we'd want them to use `package:test/scaffolding.dart`, not `package:test_api/scaffolding.dart`.

Reply:
We can put notes here saying such.

In the short term, this fixes the dartdoc warnings.

Reply:
Are there analyzer implications here? Will it start adding this import instead of `package:test/test.dart` or `package:test/scaffolding.dart`? It seems reasonable that it would, but then this isn't the behavior we want.

Reply:
Can you maybe clarify or point to some docs on exactly what this is/does?

Reply:
https://github.com/dart-lang/dartdoc/issues/2887

Sadly, there are no docs. Basically, if one member is exposed by more than one library, `dartdoc` guesses where to put it.

This "annotation" makes things explicit

Reply:
Ok, it sounds like this should be fairly low impact then and we could change it later if we decided to?

Reply:
yup